dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -62,13 +65,68 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         }
     }
 
+    // RegisterUnpackagedapplication
+    void PushNotificationManager::RegisterApplicationHelper(wil::unique_cotaskmem_string &unpackagedAppUserModelId, winrt::guid remoteId)
```

Comment:
Is remoteId the PFN for the unpackaged app? Just clarrifying

Reply:
yes pavan that is what it is.